jerry-debugger/jerry-client-ws.py

```diff
@@ -273,7 +274,19 @@ def do_step(self, args):
 
     def do_next(self, args):
         """ Next breakpoint in the same context """
-        self.exec_command(args, JERRY_DEBUGGER_NEXT)
+        if args == "":
+            args = 0
+        else:
+            try:
+                args = int(args)
+                args = min(args, len(self.debugger.last_breakpoint_hit.function.lines) -
+                           self.debugger.last_breakpoint_hit.function.line) - 1
+                self.debugger.nexts_remain = args
+            except ValueError as val_errno:
```

Comment:
I do not think it's necessary to put it all in a try block, because you only use cast in the first command.